tests/reversetest.ark

```diff
@@ -0,0 +1,11 @@
+{
+(import "Reverse.ark")
+    # --------------------------
+    #           Reverse
+    # --------------------------
+    (let reverse-tests (fun () {
```

Comment:
will need more tests, with `""` and `"a"` for example

Reply:
when I said `"a"`, i was thinking of a single character, not `"a a"`, to see if it handles well single character strings